modules/tinymce/src/plugins/insertdatetime/test/ts/browser/InsertDatetimeSanityTest.ts

```diff
@@ -1,43 +1,38 @@
-import { ApproxStructure, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { ApproxStructure } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
 
-import InsertDatetimePlugin from 'tinymce/plugins/insertdatetime/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import Plugin from 'tinymce/plugins/insertdatetime/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.insertdatetime.InsertDatetimeSanityTest', (success, failure) => {
+describe('browser.tinymce.plugins.insertdatetime.InsertDatetimeSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'insertdatetime',
+    toolbar: 'insertdatetime',
+    insertdatetime_element: true,
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin, Theme ]);
 
-  InsertDatetimePlugin();
-  SilverTheme();
+  it('TBA: Click on Insertdatetime button and select the first item from the drop down menu. Assert date time is inserted', async () => {
+    const editor = hook.editor();
 
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyUi = TinyUi(editor);
-    const tinyApis = TinyApis(editor);
+    TinyUiActions.clickOnToolbar(editor, '[aria-haspopup="true"]');
+    await TinyUiActions.pWaitForUi(editor, '[role="menu"]');
+    TinyUiActions.clickOnUi(editor, '[role="menu"] [role="menuitemcheckbox"]:first');
 
-    Pipeline.async({},
-      Log.steps('TBA', 'InsertDateTime: Click on Insertdatetime button and select the first item from the drop down menu. Assert date time is inserted', [
-        tinyUi.sClickOnToolbar('click on insertdatetime button', '[aria-haspopup="true"]'),
-        tinyUi.sWaitForUi('wait for menu', '[role="menu"]'),
-        tinyUi.sClickOnUi('click on first in menu', '[role="menu"] [role="menuitemcheckbox"]:first'),
-        tinyApis.sAssertContentStructure(ApproxStructure.build((s) => {
-          return s.element('body', {
+    TinyAssertions.assertContentStructure(editor, ApproxStructure.build((s) => {
+      return s.element('body', {
+        children: [
+          s.element('p', {
             children: [
-              s.element('p', {
-                children: [
-                  s.element('time', {})
-                ]
-              })
+              s.element('time', {})
             ]
-          });
-        }))
-      ])
-      , onSuccess, onFailure);
-  }, {
-    plugins: 'insertdatetime',
-    toolbar: 'insertdatetime',
-    insertdatetime_element: true,
-    theme: 'silver',
-    base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
-}
-);
+          })
+        ]
+      });
+    }));
+  });
+});
+
```

Comment:
I think this is an extra newline (not including the one at the EOF)

Reply:
When you open this directly it looks fine, so I think it's GitHub just being weird... https://github.com/tinymce/tinymce/blob/fdd93a23d03325bae82b57e84dba33d7a93abf11/modules/tinymce/src/plugins/insertdatetime/test/ts/browser/InsertDatetimeSanityTest.ts